zipline/gens/tradesimulation.py

```diff
@@ -78,6 +78,12 @@ def inject_algo_dt(record):
                 record.extra['algo_dt'] = self.simulation_dt
         self.processor = Processor(inject_algo_dt)
 
+    def _create_bar_data(self):
```

Comment:
Likewise, with this and the other `_create_bar_data` methods (with regard to adding a comment.)